README.md

```diff
@@ -34,7 +34,7 @@ Here are our six core certifications:
 - Responsive Web Design Principles
 - CSS Flexbox
 - CSS Grid
-** Projects **: Tribute Page, Survey Form, Product Landing Page, Technical Documentation Page, Personal Portfolio Webpage
+  ** Projects **: Tribute Page, Survey Form, Product Landing Page, Technical Documentation Page, Personal Portfolio Webpage
```

Comment:
These all still need the new line before them **and** the spaces should be removed around "Projects". Please review [Markdown syntax](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) if needed.

Below is more specifically what we mean by "remove spaces around 'Projects'":

```diff
- ** Projects **: Tribute Page...
+ **Projects**: Tribute Page...
```

Reply:
Thank you for the feedback and being patient.  I removed the spaces around the 'projects' like you described (I hope) and added a new line before projects.  Hopefully this is what you mean.